routes/users.js

```diff
@@ -1,10 +1,54 @@
 const express = require('express');
+const pool = require('../connection/connection');
 
 const router = express.Router();
 
-/* GET users listing. */
+// Get Method
```

Comment:
Everything is ok here but make sure to add the second GET method with the `/:id`